packages/@okta/vuepress-site/docs/reference/index.md

```diff
@@ -2,21 +2,30 @@
 title: Reference overview
 ---
 
-# Reference overview
+# Reference 
 
 Details on parameters, requests, and responses for Okta's API endpoints.
 
-Okta APIs generally fall into one of two categories: signing in your users or managing
-resources in your Okta org.
 
-## Sign in your Users
-API endpoints to authenticate your users, challenge for factors, recover passwords, and more.
-  - The [Authentication Api](/docs/reference/api/authn) controls user accesss to Okta.
-  - The [OpenID Connect & OAuth 2.0 API](/docs/concepts/oauth-openid) controls users access to your applications.
 
-[Learn about which APIs to use.](/docs/concepts/oauth-openid/#authentication-api-vs-oauth-20-vs-openid-connect)
+## Core Okta API
+The Core Okta API is the primary way that apps and services interact with Okta. You can use it to implement basic auth functions such as signing in your users and programmatically managing your Okta objects. 
 
-## Manage Okta objects
-REST endpoints configure resources such as users, apps, sessions, and factors when needed.
+[Explore core Okta API](/docs/reference/core-okta-api/)
 
-For example, see the [Users API](/docs/reference/api/users) for CRUD operations on users.
+<Cards><Card href="/docs/concepts/authentication/" :showHeaderIcon=false cardTitle="Postman Collections" :showFooter=true>Import any Okta API collection for Postman.</Card><Card href="/docs/concepts/oauth-openid/" :showHeaderIcon=false cardTitle="Advanced Service Access API" :showFooter=true>Scale your control of servers with automation.</Card></Cards>
+
+<HorizontalDivider/>
```

Comment:
We don't use horizontal dividers anywhere else in the site. Is there a reason that these were added here and in line 26?

Reply:
It was present on designs here in Figma, so added them. 

https://www.figma.com/file/2ebFMRMittnBfnuOtNEJ3T/API-Reference-and-Release-Notes-(Dev-Doc)-IA-Clean-Up?node-id=1017%3A0

Reply:
Whose figma design is that - so can we discuss with the design/UX person?